tensorflow_addons/layers/spectral_normalization.py

```diff
@@ -26,17 +26,21 @@ class SpectralNormalization(tf.keras.layers.Wrapper):
 
     See [Spectral Normalization for Generative Adversarial Networks](https://arxiv.org/abs/1802.05957).
 
-    ```python
-    net = SpectralNormalization(
-        tf.keras.layers.Conv2D(2, 2, activation="relu"),
-        input_shape=(32, 32, 3))(x)
-    net = SpectralNormalization(
-        tf.keras.layers.Conv2D(16, 5, activation="relu"))(net)
-    net = SpectralNormalization(
-        tf.keras.layers.Dense(120, activation="relu"))(net)
-    net = SpectralNormalization(
-        tf.keras.layers.Dense(n_classes))(net)
-    ```
+    >>> x = tf.constant( [[ 177,  164,  169,  165,  133,  134,  168,  174,  160,  165], [154,  146,  100,   54,   48,   19,    8,   6,
+    ... 4,  148], [162,  152,   94,  200,  177,   73,   27,    1,  145,  153], [186,   56,  177,  202,  176,  132,   51,
+    ... 13,   43,  152], [186,   17,  151,  162,  119,  106,   70,   11,   25,  100], [146,   12,  154,  141,  140,  146,
+    ... 92,   21,   16,  101],  [93,    9,  192,  150,  148,  148,   42,   26,   19,   92], [ 91,   13,  125,  129,  114,
+    ... 127,   42,   27,    6,   86], [113,   43,    9,  190,  125,   58,   38,   26,   12,   99], [134,   29,   19,   17,
+    ... 72,   42,   61,   34,   12,   83]],dtype = tf.int32)
```

Comment:
The shape of `x` does not work for this example. Also I think if we are going to specify `x`, better use something like `tf.random.normal(shape=(1,10,10,1))` instead of defining explicitly this tensor

Reply:
+1

Reply:
@charlielito. Thanks a lot for the suggestion it worked and saved me a lot of time. Can you please suggest what value should n_classes get.